src/main/resources/hudson/plugins/nextexecutions/NextExecutionsWidget/index.jelly

```diff
@@ -2,12 +2,19 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 <j:if test="${it.showWidget()}">
   <l:pane width="2" title="${it.widgetName}" id="${it.widgetId}">
-    <j:forEach var="w" items="${it.builds}">
+    <j:if test="${it.builds() == null}"> 
       <tr>
-        <td class="pane"><a tooltip="${w.name}" href="${w.url}">${w.name}</a></td>
-        <td class="pane" tooltip="${w.timeToGo}">${w.date}</td>
+        <td class="pane">No execution in process.</td>

```

Comment:
This must support i18n and added on `https://github.com/jenkinsci/next-executions-plugin/blob/main/src/main/resources/hudson/plugins/nextexecutions/Messages.properties`

I think also better phrasing would be `No next executions found.`

Reply:
Sorry, I'm a completely new to jelly script. Didn't realize that I would also have to add it to https://github.com/jenkinsci/next-executions-plugin/blob/main/src/main/resources/hudson/plugins/nextexecutions/Messages.properties.

Also, I was trying to find some sources from where I can learn more about jelly script but didn't find much online and the information was in bits and pieces. Do you have any recommendation or good sources from where I can learn more about it?

Reply:
https://www.jenkins.io/doc/developer/internationalization/i18n-jelly-views/

Reply:
Thanks a lot.